pom.xml

```diff
@@ -366,17 +366,30 @@
     </resources>
     <plugins>
       <plugin>
-        <groupId>org.apache.tomcat.maven</groupId>
-        <artifactId>tomcat7-maven-plugin</artifactId>
-        <version>2.2</version>
+        <groupId>org.codehaus.cargo</groupId>
```

Comment:
There is no maven-tomcat8-plugin. This seems to be the preferred solution. It means you need to use `tomcat cargo:run` to run from the command line in your local dev environment.